app/apollo/resolvers.ts

```diff
@@ -1,7 +1,9 @@
 // Write resolvers to respond to our queries and mutations
+// https://www.notion.so/neo4j-graphql-v1-0-0-alpha-2-d47908030d4e4a0c86babbaef63887d0
 export const resolvers = {
   Query: {
-    hello: (_parent, _args, _context) =>
-      `Hello. The current timestamp is ${Date.now()}`,
+    async hello(_parent, _args, _context) {
```

Comment:
I know, I know...this particular function doesn't need to be async. I have it as `async` for reference only, since practically every other custom resolver will likely be async and can quickly borrow this pattern.